org.kavil.gestimator.json

```diff
@@ -0,0 +1,106 @@
+{
+    "app-id": "org.kavil.gestimator",
```

Comment:
Is this app-id correct? There doesn't seem to be a domain registered for kavil.org

Reply:
Sorry the domain given is not registered. Have now registered kavilgroup.com. Will resubmit merge request after making correction to the app id as com.kavilgroup.gestimator. Sorry for the inconvenience caused. I will close this pull request now.

Reply:
You didn't need to close the PR, you can just push a new commit if you wish